components/PsychrometricCharts.py

```diff
@@ -0,0 +1,177 @@
+import math
+import numpy as np
+import plotly.graph_objs as go
+from dash import Dash, dcc, html
+from dash.dependencies import Input, Output
+
+# 计算函数
+def calculate_saturation_vapor_pressure(temp_c):
+    return 610.94 * np.exp(17.625 * temp_c / (temp_c + 243.04))
```

Comment:
this is good but you should use the function from pythermalcomfort so we do not have code repetitions

Reply:
got it,sir i was working on that and almost done